homeassistant/components/websocket_api/connection.py

```diff
@@ -31,6 +31,16 @@ def context(self, msg):
             return Context()
         return Context(user_id=user.id)
 
+    @callback
+    def send_result(self, msg_id, result=None):
```

Comment:
For curiosity, why wasn't this needed before this integration?

Reply:
It's a helper. In other places we always call `connection.send_message(websocket_api.result_message(msg_id, result))`